### PR TITLE
Added arp flood mode to disable proxy-arp

### PR DESCRIPTION
**Pull request by gkvijay**

### CODE REVIEW

File: ofnetAgent.go [hunk 1]
```diff
@@ -127,16 +128,17 @@ const (
 // Create a new Ofnet agent and initialize it
 /*  routerInfo[0] -> Uplink nexthop interface
  */
-func NewOfnetAgent(bridgeName string, dpName string, localIp net.IP, rpcPort uint16,
```
Comment (abhinandanpb) [inline ofnetAgent.go]: are we expecting the knob to diruptive ? Meaning once you set the arp mode we can change it ? We could make it configurable ? which would mean not set the arp mode during ofnet agent creation.

Comment (gkvijay) [inline ofnetAgent.go]: Based on discussion with @abhinandanpb the restart of network driver is currently requiring all the configs on the network to be deleted during the fwd mode change. This is probably not required for arp mode change. Will make that change

File: ofnetAgent.go [hunk 2]
```diff
@@ -137,6 +138,7 @@ func NewOfnetAgent(bridgeName string, dpName string, localIp net.IP, rpcPort uin
 	agent.MyPort = rpcPort
 	agent.MyAddr = localIp.String()
 	agent.dpName = dpName
+	agent.arpMode = "proxy"
```
Comment (DivyaVavili) [inline ofnetAgent.go]: Minor fix: Could we have constants for this? For understanding, it's better to have constants list down all the supported modes in one place than check it all over the code.

File: vlanBridge.go [hunk 3]
```diff
@@ -219,6 +224,24 @@ func (vl *VlanBridge) sendGARPAll() {
...
+// Update global config
+func (vl *VlanBridge) GlobalConfigUpdate(cfg OfnetGlobalConfig) error {
+	if vl.agent.arpMode == cfg.ArpMode {
+		log.Errorf("no change in ARP mode %s", vl.agent.arpMode)
+	} else {
+		vl.agent.arpMode = cfg.ArpMode
+		add := (vl.agent.arpMode == "proxy")
+		vl.updateArpRedirectFlow(add)
+
+		if cfg.ArpMode == "flood" {
```
Comment (gkvijay) [inline vlanBridge.go]: Not changing the function name as it is used during the init as well. But moved the trigger GARP into the update function. @DivyaVavili PTAL

**Pull request merged**